src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working.");

```

Comment:
Similarly, the stop message could be more informative. Instead of 'Bulldozer stopped working.', you could use 'Bulldozer has finished pushing earth.' to provide more context.